model/post_test.go

```diff
@@ -884,3 +884,44 @@ func TestSearchParameterFromJson(t *testing.T) {
 		require.Equal(t, "test", *params.Terms)
 	})
 }
+
+func TestPostAttachments(t *testing.T) {
+	p := &Post{
+		Props: map[string]interface{}{
+			"attachments": []byte(`[{
+				"actions" : {null}
+			}]
+			`),
+		},
+	}
+
+	t.Run("empty actions", func(t *testing.T) {
+		p.Props["attachments"] = []interface{}{
+			map[string]interface{}{"actions": []interface{}{}},
+		}
+		attachments := p.Attachments()
+		require.Empty(t, attachments[0].Actions)
+	})
+
+	t.Run("a couple of actions", func(t *testing.T) {
+		p.Props["attachments"] = []interface{}{
+			map[string]interface{}{"actions": []interface{}{
+				map[string]interface{}{"id": "test1"}, map[string]interface{}{"id": "test2"}},
+			},
+		}
+
+		attachments := p.Attachments()
+		require.Len(t, attachments[0].Actions, 2)
```

Comment:
I understand the filtering is rather straightforward but would it make sense to also check the expected values, just to be extra confident? 

Same would apply for the case below.

Reply:
Fair enough 👍